test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Comparers/OpenApiSecurityRequirementComparerTests.cs

```diff
@@ -0,0 +1,73 @@
+using System.Dynamic;
+using System.ComponentModel;
+using System;
+
+using System.Collections.Generic;
+using System.Linq;
+
+using Microsoft.OpenApi.Models;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests.Fakes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Comparers;
+using FluentAssertions;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests.Comparers
+{
+    [TestClass]
+    public class OpenApiSecurityRequirementComparerTests
+    {
+        private OpenApiSecurityRequirementComparer _comparer;
+
+        private OpenApiSecurityRequirement initializeRequirement(string referenceId)
+        {
+            var reference = new OpenApiReference { Id = referenceId };
+            var scheme = new OpenApiSecurityScheme { Reference = reference };
+            return new OpenApiSecurityRequirement { { scheme, null } };
+        }
```

Comment:
Could you move this private method to the bottom of the class? In addition to that, the method name should be PascalCasing

Reply:
I have made the necessary changes as requested. I moved the private method to the bottom of the class and renamed it using PascalCasing.